styles/mobile.css

```diff
@@ -0,0 +1,1305 @@
+    max-width: 23%;
+  }
+
+  .latestdeals .cards:last-child {
+    padding-right: 0;
+  }
+}
+
+/* ########################## */
+
+/* ##### FOOTER SECTION ##### */
+
+/* ########################## */
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(32, 32, 32);
+  height: 160px;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+.bottombar {
+  background-color: rgb(24, 24, 24);
+  height: 120px;
+  width: 100%;
+}
+
+.social {
+  margin-top: 10px;
+  height: 50px;
+  min-width: 400px;
+}
+
+.social ul {
+  display: flex;
+  justify-content: space-around;
+}
+
+.social li {
+  background-color: rgb(66, 66, 66);
+  background-size: 45%;
+  border-radius: 30px;
+  height: 40px;
+  width: 40px;
+}
+
+.links {
+  margin-top: 10px;
+  height: 50px;
+  width: 300px;
+}
+
+.links ul {
+  display: flex;
+  justify-content: space-between;
+}
+
+.links li a {
+  font-family: 'Shentox', sans-serif;
+  font-size: 0.9rem;
+  list-style: none;
+  text-decoration: none;
+  color: rgb(206, 206, 206);
+}
+
+.social-icon {
+  width: 20px;
+  padding: 10px;
+}
+
+.bottombar div {
+  font-family: 'ARS Maquette', 'Helvetica', Georgia, 'Times New Roman', Times, serif;
+  font-size: 0.8rem;
+  height: 100%;
+  text-align: center;
+  color: gray;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.bottom-icon {
+  height: 10px;
+}
+
+@media only screen and (min-width: 768px) {
+  .dashboard {
+    height: 75vh;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    grid-gap: 10px;
+  }
+
+  .dashboard .cards {
+    margin: 0;
+  }
+
+  .main-1 .cards-title {
+    font-size: 2rem;
+  }
+
+  .main-title {
+    visibility: hidden;
+    width: 95%;
+    margin: 6% 2%;
+    display: flex;
+    justify-content: space-between;
+    font-size: 1rem;
+  }
+
+  .main-title img {
+    height: 80px;
+  }
+
+  .main-1 {
+    background:
+      linear-gradient(to top, black, rgba(255, 255, 255, 0)),
+      url(./../assets/images/header-image-microsoft-office-ipad-apps.png) 50% 50% no-repeat;
+    grid-column: 1/3;
+    grid-row: 1/3;
+  }
+
+  .main-2 {
+    grid-row: 1;
+    grid-column: 3;
+  }
+
+  .main-3 {
+    grid-row: 2;
+    grid-column: 3;
+  }
+
+  .posts article {
+    width: 47%;
+    margin: 10px;
+  }
+
+  .posts .cards {
+    height: 15rem;
+  }
+
+  .lat-li li {
+    font-size: 0.8rem;
+
+    /* margin: 20px 20px; */
+    color: black;
+  }
+
+  .latest {
+    font-size: 0.8rem;
+  }
+}
+
+@media only screen and (min-width: 1024px) {
+  .dashboard {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    grid-gap: 10px;
+  }
+
+  .main-1 .cards-title {
+    font-size: 3rem;
+  }
+
+  .main-title {
+    visibility: visible;
+    width: 60%;
+    margin: 6% 2%;
+    position: absolute;
+    top: 50px;
+    display: flex;
+
+    /* justify-content: space-between; */
```

Comment:
- [ ] Please remove this commented piece of code.

Reply:
Fixed!